docs.html

```diff
@@ -939,6 +940,55 @@ <h3 class="component-variant-heading">Grids with three columns.</h3>
                     style="width: 742px; height: 468px; border:0; transform: scale(1); overflow:hidden;"

```

Comment:
![cl review](https://user-images.githubusercontent.com/66775072/154851386-a58567ab-30d4-4cfb-a4d7-1b77c1f210f2.png)


Don't use inline stying over here. Instead, style using classnames. I have seen that all code snippets are of different height and width. Just a suggestion that all code snippets should have the same width and height. Wherever needed use scrollbar there.

Reply:
Yes, there is inconsistency. Styling these code snippets from carbon have been difficult, so I'm planning on replacing all the code snippets with some other means after the submission as it will take a lot of time to refactor everything. Thank you for pointing this out, will keep in mind !